chain-crypto/src/key.rs

```diff
@@ -19,6 +19,9 @@ pub trait AsymmetricKey {
     type Secret: AsRef<[u8]> + Clone;
     type Public: AsRef<[u8]> + Clone + PartialEq + Eq + Hash;
 
+    const SECRET_BECH32_HRP: &'static str;
+    const PUBLIC_BECH32_HRP: &'static str;
```

Comment:
what HRP stands for ?

Reply:
Human readable part. This name and its abbreviation seem standard in bech32 nomenclature.

Reply:
thanks